src/test/lang/generator/deep_copy_test.cpp

```diff
@@ -37,7 +37,7 @@ TEST(lang, run_deep_copy) {
   std::stringstream model_output;
 
   // instantiate good test model parser-generator/deep-copy-warning
-  stan_model model(empty_data_context, &model_output);
+  stan_model model(empty_data_context, 0, &model_output);
```

Comment:
stanc3 only generates the constructor for 3 arguments. Stanc2 also generated a constructor without the seed, making it 0 by default. As this was only used for tests, I think it makes more sense to just explicitly set it here. 

The alternative would be to add the same constructor in stanc3 also. I dont think that is necessary but will leave that up to the reviewer.